ttkthemes/themed_tk.py

```diff
@@ -44,13 +44,21 @@ def __init__(self, *args, **kwargs):
         # Set initial theme
         if theme is not None and theme in self.get_themes():
             self.set_theme(theme, toplevel, background)
+        elif theme is None and background is not None:
```

Comment:
```suggestion
        elif theme is None and background is not None and theme in self.get_themes():
```
Even though all themes included with the `ttkthemes` package are now available on all platforms, there are still platform-specific platforms that will throw errors when they are referenced on platforms where they are not available. The `theme` keyword argument provides an error-free silent-fail method of setting a theme upon initialization, thus this check is essential.